docs/data/charts/brush/brush.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Charts - Brush
+productId: x-charts
+components: ChartsBrushOverlay
+---
+
+# Charts - Brush
+
+<p class="description">The brush interaction allows users to select a region on the chart by clicking and dragging.</p>
+
+## Overview
+
+The brush feature provides a way to track user selections on charts through a click-and-drag interaction.
+It captures the start and current positions of the user's selection, which can be used for various purposes such as:
+
+- Highlighting specific areas
+- Creating custom interactions
+
+The brush is available in the `LineChart`, `BarChart`, and `ScatterChart` types and provides visual feedback through the `ChartsBrushOverlay` component.
+
+## Basic usage
+
+To display visual feedback when users interact with the chart, enable the brush with `brushConfig={{ enabled: true }}` and add the `ChartsBrushOverlay` component as a child of your chart.
+
+The brush itself has no default behavior; you can use the `useBrush` hook to create custom interactions based on the brush state.
+
+{{"demo": "BrushBasic.js"}}
+
+## Customization examples
+
+### Custom overlay
+
+You can create a custom brush overlay by building your own component that uses the `useBrush` hook.
+This example shows how to display the values at the start and end positions, along with the difference and percentage change between them.
+
+{{"demo": "BrushCustomOverlay.js"}}
```

Comment:
This demo is 🔥 and can help users quickly compare %change. It will also be helpful, if we add the x-axis date range in the brush overlay label, like shown in the issue video?